tests/pass-dep/concurrency/linux-futex.rs

```diff
@@ -35,17 +35,6 @@ fn wake_nobody() {
     }
 }
 
-fn wake_dangling() {
```

Comment:
This should still work. At least parking_lot depends on "wake after free" to be acceptable.